### PR TITLE
Document array update operator

### LINKED ISSUE #2157: Add documentation for update operators
Add new sections for update operators in the documentation

- [x] Field update operators
- [x] Array update operators

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #2157.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/docs/operators/update_operators/array-update-operators.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+);
+```
+
+After the operation, the updated document looks like this:
+
+```js
+[ {
+    _id: 1,
+    items: [ 'pens', 'pencils', 'paper', 'erasers', 'rulers', 'markers' ]
+    }]
```
Comment (chilagrow) [inline website/docs/operators/update_operators/array-update-operators.md:44]: The formatting is a bit weird here.

File: website/docs/operators/update_operators/array-update-operators.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+```
+
+## $addToSet
+
+The `$addToSet` operator updates an array by adding a specified element to an array if the elenent does not already exist in the array.
```
Comment (chilagrow) [inline website/docs/operators/update_operators/array-update-operators.md:49]:
```suggestion
The `$addToSet` operator updates an array by adding a specified element to an array if the element does not already exist in the array.
```

File: website/docs/operators/update_operators/array-update-operators.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+);
+```
+
+The updated now looks like this:
+
+```js
+[ {
+    _id: 1,
+    items: [ 'pencils', 'paper', 'erasers' ]
+    }]
```
Comment (chilagrow) [inline website/docs/operators/update_operators/array-update-operators.md:159]: Formatting is also strange here

File: website/docs/operators/update_operators/array-update-operators.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+
+Insert the following document into a `fruits` collection:
+
+```js
+db.fruits.insertMany([
+    { _id: 1, fruits: [
+        { type: "apple", color: "red" },
+        { type: "banana", color: "yellow" },
+        { type: "orange", color: "orange" }
+        ] }
```
Comment (chilagrow) [inline website/docs/operators/update_operators/array-update-operators.md:202]: Also here.

**Review by chilagrow — COMMENTED** [review]
Looks good, just formatting issues.

**Review by chilagrow — APPROVED** [review]

File: website/docs/operators/update_operators/array-update-operators.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+## $pullAll
+
+The `$pullAll` operator removes all the matching elements in a specified query from an array.
+
+Insert the following document into a `store` collection:
+
+```js
+db.store.insertMany([
+  { _id: 1, items: ["pens", "pencils", "paper", "erasers", "rulers"] },
```
Comment (rumyantseva) [inline website/docs/operators/update_operators/array-update-operators.md:176]: Should we have some duplicates (e.g. `paper` mentioned twice) to demonstrate how duplicate elements work with `$pullAll`?

**Review by rumyantseva — APPROVED** [review]
The documentation looks good to me. It's easy to read and understand, and I like the examples. It covers the array update operators we currently support.

We don't mention anywhere that we will support more array update operators in the future. Should we mention it?

**Comment by rumyantseva** [comment]
@w84thesun please review! As you are the main author of array update operators, your review is highly appreciated!

**Review by ptrfarkas — APPROVED** [review]

**Pull request merged**